graphs/find_connected_components.cpp

```diff
@@ -0,0 +1,65 @@
+#include "../headers.h"
+
+/* Get connected components in a graph */
+
+void get_connected_components(vector<vi>& g, int n)
+{
+	/* Creating the graph */
+	map<int, set<int>> graph;
+	for(auto c: g)
+	{
+		graph[c[0]].insert(c[1]);
+		graph[c[1]].insert(c[0]);
+	}
+
+	vector<set<int>> result;
+	std::vector<int> visited(n, false);
```

Comment:
consider using vector of bool, to avoid casting

Reply:
Makes complete sense! :) Thanks!